docs/guides/references/cypress-studio.mdx

```diff
@@ -7,7 +7,7 @@ title: Cypress Studio
 <strong class="alert-header"><Icon name="exclamation-triangle"></Icon>
 Removed</strong>
 
-Cypress Studio has been removed in Cypress v10 and will be rethought/revisited
+Cypress Studio has been removed in Cypress 10 and will be rethought/revisited
```

Comment:
😆 this was a hot topic at one point and we made all refs "v10". Did that change again?

Reply:
ya dont think we are suppose to use 'v' in front of the versions. I  asked Matt V for clarificaiton.